diff -ur postgresql-9.2.4/configure postgresql-9.2.4.fixed/configure
--- postgresql-9.2.4/configure	2013-06-19 16:14:52.000000000 -0400
+++ postgresql-9.2.4.fixed/configure	2013-06-19 16:11:52.000000000 -0400
@@ -8424,7 +8424,7 @@
 else	READLINE_ORDER="-ledit -lreadline"
 fi
 for pgac_rllib in $READLINE_ORDER ; do
-  for pgac_lib in "" " -ltermcap" " -lncurses" " -lcurses" ; do
+  for pgac_lib in "" " -lncurses" " -lcurses" ; do
     LIBS="${pgac_rllib}${pgac_lib} $pgac_save_LIBS"
     cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
